wiki/Contests/KPMC/en.md

```diff
@@ -0,0 +1,5 @@
+# K-Pop Mapping Contest
+
+Index page for all the K-Pop Mapping Contest series.
```

Comment:
grammar (also we have a [template for this](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE-SERIES.md))

```suggestion
Index page for all of the K-Pop Mapping Contest series.
```